Dockerfile

```diff
@@ -0,0 +1,219 @@
+	; \
+	pecl clear-cache; \
+	docker-php-ext-enable \
+		apcu \
+		opcache \
+	; \
+	\
+	runDeps="$( \
+		scanelf --needed --nobanner --format '%n#p' --recursive /usr/local/lib/php/extensions \
+			| tr ',' '\n' \
+			| sort -u \
+			| awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 { next } { print "so:" $1 }' \
+	)"; \
+	apk add --no-cache --virtual .sylius-phpexts-rundeps $runDeps; \
+	\
+	apk del .build-deps
+
+ARG BLACKFIRE=false
+
+RUN set -eux; \
+	if [ ! -z "$BLACKFIRE" ] && [ "$BLACKFIRE" = "true" ]; then \
+		curl -A "Docker" -o /tmp/blackfire-probe.tar.gz -D - -L -s https://blackfire.io/api/v1/releases/probe/php/alpine/amd64/$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;"); \
+		mkdir -p /tmp/blackfire; \
+		tar zxpf /tmp/blackfire-probe.tar.gz -C /tmp/blackfire; \
+		mv /tmp/blackfire/blackfire-*.so $(php -r "echo ini_get ('extension_dir');")/blackfire.so; \
+		printf "extension=blackfire.so\nblackfire.agent_socket=tcp://blackfire:8707\n" > $PHP_INI_DIR/conf.d/blackfire.ini; \
+		rm -rf /tmp/blackfire /tmp/blackfire-probe.tar.gz; \
+		mkdir -p /tmp/blackfire; \
+		curl -A "Docker" -L https://blackfire.io/api/v1/releases/client/linux_static/amd64 | tar zxp -C /tmp/blackfire; \
+		mv /tmp/blackfire/blackfire /usr/bin/blackfire; \
+		rm -Rf /tmp/blackfire; \
+	fi
+
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+# https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
+ENV COMPOSER_ALLOW_SUPERUSER=1
+RUN set -eux; \
+	composer global require "hirak/prestissimo:^0.3" --prefer-dist --no-progress --no-suggest --classmap-authoritative; \
+	composer clear-cache
+ENV PATH="${PATH}:/root/.composer/vendor/bin"
+
+WORKDIR /srv/sylius
+
+RUN git clone --depth 1 --single-branch --branch "1.7" https://github.com/Sylius/Sylius-Standard.git .
+
+RUN set -eux; \
+	cp docker/php/php.ini /usr/local/etc/php/php.ini; \
+	cp docker/php/php-cli.ini /usr/local/etc/php/php-cli.ini
+
+COPY . themes/BootstrapTheme
+
+# build for production
+ARG APP_ENV=prod
+
+RUN set -eux; \
+	sed -i s/2G/6G/g /usr/local/etc/php/php-cli.ini; \
+	composer install --prefer-dist --no-autoloader --no-scripts --no-progress --no-suggest; \
+	composer require symfony/webpack-encore-bundle; \
+	composer clear-cache
+
+COPY docker/assets.yaml config/packages/assets.yaml
+COPY docker/webpack_encore.yaml config/packages/webpack_encore.yaml
+
+COPY docker/SyliusAdminBundle templates/bundles/SyliusAdminBundle
+COPY docker/SyliusShopBundle templates/bundles/SyliusShopBundle
+
+
+FROM node:${NODE_VERSION}-alpine AS sylius_nodejs
+
+WORKDIR /srv/sylius
+
+RUN set -eux; \
+	apk add --no-cache --virtual .build-deps \
+		g++ \
+		gcc \
+		git \
+		make \
+		python \
+	;
+
+# prevent the reinstallation of vendors at every changes in the source code
+COPY --from=sylius_php_base /srv/sylius/package.json ./
+COPY --from=sylius_php_base /srv/sylius/yarn.lock ./
+
+RUN set -eux; \
+	yarn add \
+		@symfony/webpack-encore \
+		sass-loader@^8.0.0 \
```

Comment:
Use ^9.0.0 instead of ^8.0.0

Reply:
Okay, but I guess the instructions in the README and the docs need to be updated, as it states to do the following:
`yarn add @symfony/webpack-encore sass-loader@^7.0.0 node-sass lodash.throttle -D`
https://github.com/Sylius/BootstrapTheme/blob/master/README.md
https://docs.sylius.com/en/latest/book/themes/bootstrap-theme.html

Reply:
Yes, I will made a PR asap.